mesh/README.md

```diff
@@ -0,0 +1,48 @@
+# MESH Command Line Tool
+
+## Setup
+
+Copy `env.example.sh` to `env.sh` and fill in the values as described in the comments and below.
+
+You **MUST** provide files containing **your** OpenTest endpoint private key and endpoint certificate.
+These can be found in your OpenTest welcome email. Copy the values into either the path/files used by default
+(see `env.example.sh` for instruction) or to paths/files of your choosing (provided you set the variables
+accordingly)
+
+## Usage
+
+### Authenticate
+
+MESH API docs says to do this first but our experience seems that its not needed.
+
+    ./mesh.sh auth
+    
+### List Inbox Messages
+
+List all messages in the inbox. The JSON response is printed to the console.
```

Comment:
It's first 500, not all messages. Also it lists only message ids.

Reply:
fixed by just referring to mesh api docs